rstudio/Dockerfile

```diff
@@ -34,8 +35,8 @@ RUN /usr/local/lib/R/site-library/littler/examples/installBioc.r \
     org.Hs.eg.db \
     pathview
 
-# Install synapser using `install.packages()` (error when installed with install2.r)
-RUN R -e "install.packages('synapser', repos=c('http://ran.synapse.org', 'http://cran.fhcrc.org'))"
+# Install synapser directly from source
+RUN R -e "remotes::install_github(repo='https://github.com/generalui/synapser/tree/reticulate', ref='reticulate')"
```

Comment:
@vpchung you should be able to install from the Sage branch, or... even that last command should work.

Reply:
Hm, I wasn't able to get it working before.  But that was also before I updated the base image version...

I'll try rebuilding with `RUN R -e "install.packages('synapser', repos=c('http://ran.synapse.org', 'http://cran.fhcrc.org'))"` to check.

Reply:
Ah okay, you were right! I only needed to update the base image version. Reverting this line change.